packages/grafana-ui/package.json

```diff
@@ -59,6 +59,7 @@
     "memoize-one": "6.0.0",
     "moment": "2.29.1",
     "monaco-editor": "^0.31.1",
+    "ol": "6.12.0",
```

Comment:
we already depend on open layers in `@grafana/data` -- but only the types are used.

Reply:
I guess then @grafana/data could depend on https://www.npmjs.com/package/@types/ol instead of entire lib? WDYT?

Reply:
Assuming that package is identical to the the types published in "ol" then yes... but  I am not sure if they are the same.  For sure the latest versions are not in sync -- so i don't really want to change that unless it is necessary